justfile

```diff
@@ -0,0 +1,71 @@
+
+target := "x86_64-unknown-linux-musl"
+profile := 'debug' # or 'release'
+_just-cargo := 'just-cargo profile=' + profile +  ' target=' + target
```

Comment:
out of curiosity, what does `just-cargo` do?

Reply:
See the linked dev PR https://github.com/linkerd/dev/pull/13. Basically it ensure all of the proper environment variables are set for cross-compilation and it passes command output through `cargo-action-fmt` when appropriate

Reply:
ah, that's cool